src/pages/Checkout/CheckoutPage.vue

```diff
@@ -551,19 +543,10 @@ export default {
 	},
 	methods: {
 		loginToContinue(event) {
-			if (this.redirectToLoginExperimentVersion) {
-				this.$kvTrackEvent(
-					'Basket',
-					'EXP-GROW-282-Oct2020',
-					this.redirectToLoginExperimentVersion,
-				);
-			}
-
-			if (this.redirectToLoginExperimentVersion !== 'b' && this.kvAuth0.enabled) {
+			if (this.kvAuth0.enabled) {
 				event.preventDefault();
 				this.doPopupLogin();
```

Comment:
@emuvente @edwinf-barreram  I think we can just set `event.preventDefault();` as the only statement in `loginToContinue` making sure to remove the `this.doPoupLogin` method call as that would bring back the pop up we're removing.